docs/README.md

```diff
@@ -20,81 +20,160 @@
 The website is hosted at http://mxnet.incubator.apache.org/.
 http://mxnet.io redirects to this site and advised to use links with http://mxnet.incubator.apache.org/ instead of http://mxnet.io/.
 
-MXNet Documentation Website is built with [Sphinx](http://www.sphinx-doc.org) and a variety of plugins including [pandoc](https://pandoc.org/), [recommonmark](https://github.com/rtfd/recommonmark), a custom Sphinx plugin ([mxdoc.py](https://github.com/apache/incubator-mxnet/blob/master/docs/mxdoc.py)).
+## Website & Documentation Contributions
 
+Detailed information on website development, continuous integration, and proposals for future projects can be found on the [MXNet Wiki](https://cwiki.apache.org/confluence/display/MXNET/Website).
+
+The website is built using Jekyll. You may run your own version of the static website by following the instructions on the wiki.
+
+Each language documentation is built in a modular way, so that if you are a contributor to Julia, for example, you only need Julia-related tools to build it. Each language API has a section on installation and building along with how to build the docs locally.
+
+You can also use the project's CI tools to emulate any changes with Docker. You can use these tools to install dependencies and run the parts of the build you want to test.
+
+If you plan to contribute changes to the documentation or website, please submit a pull request. Contributions are welcome!
+
+## Python Docs
+
+MXNet's Python documentation is built with [Sphinx](http://www.sphinx-doc.org) and a variety of plugins including [pandoc](https://pandoc.org/), and [recommonmark](https://github.com/rtfd/recommonmark).
+
+More information on the dependencies can be found in the [CI folder's installation scripts](https://github.com/apache/incubator-mxnet/tree/master/ci/docker/install/ubuntu_docs.sh).
```

Comment:
nit: when i look up the file https://github.com/apache/incubator-mxnet/blob/master/ci/docker/install/ubuntu_docs.sh , it says ```pip3 install -r /work/docs_requirements```. maybe we should make it clear where this is coming from. either in the script or here.